packages/passport/sdk-sample-app/package.json

```diff
@@ -34,8 +34,8 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "d": "next dev",
-    "dev-with-sdk": "concurrently 'yarn workspace @imtbl/passport-sdk-sample-app dev' 'yarn workspace @imtbl/passport dev'",
+    "dev": "next dev",
```

Comment:
Just double check for me but I think the `dev` script in `passport/sdk/package.json` is also `d` by mistake as well?

Reply:
yes. because "next dev" is itself a watch script already.